.evergreen/csfle/azurekms/create-and-setup-vm.sh

```diff
@@ -26,8 +26,6 @@ fi
 
 # Set defaults.
 export AZUREKMS_IMAGE=${AZUREKMS_IMAGE:-"Debian:debian-11:11:0.20221020.1174"}
-# Install az.
```

Comment:
Suggest adding a check to verify the presence of the `az` program (with required version) to validate the conditions documented in the `azurekms/README.md` file. This may help detect and encourage updating tasks that use incompatible distros.

Reply:
I agree that may be helpful. baddfd44dd140551a5483a91bd0d37d665ef9096 adds a check.